client/app/bundles/buttons/ButtonFilled.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import css from './button.scss';
+
+class ButtonFilled extends React.Component {
+   render() {
+       
+       
+      return (
+          <div>
```

Comment:
The `<div>` wrappers could be omitted, I think, so this component could be used inline as well!

Reply:
I believe you have to use `fragments` to omit the `<div>` wrappers?
https://reactjs.org/blog/2017/11/28/react-v16.2.0-fragment-support.html#jsx-fragment-syntax

Reply:
@pederjohnsen Fragments is a nicer way of returning an array of HTML elements, but it's not necessary here because we're returning a single <button> element.

Reply:
@baohouse nice to know :) 👍